src/OpenTelemetry/Trace/ActivityExporter.cs

```diff
@@ -100,7 +104,9 @@ public void Dispose()
         /// wait indefinitely.
         /// </param>
         /// <remarks>
-        /// This function should not throw exception.
+        /// This function is called synchronously on the thread which made the
+        /// first call to <c>Shutdown</c>. This function should not throw
+        /// exception.
```

Comment:
nit: Something seems a bit off about the last sentence. `should not throw an exception` or `should not throw exceptions`?

Reply:
Nice catch! I've changed to `exceptions`.